lhungtam/server.c

```diff
@@ -0,0 +1,132 @@
+/*
+ * server.c
+ *
+ *  Created on: May 31, 2019
+ *      Author: lhungtam
+ */
+
+
+#include <sys/socket.h>
+#include <sys/select.h>
+#include <stdio.h>
+#include <netinet/in.h>
+#include <arpa/inet.h>
+#include <malloc.h>
+#include <string.h>
+#include <sys/time.h>
+#include <unistd.h>
+#include <errno.h>
+
+int client[10], sizeMaxClient=10;
+
+void clearClient(){
```

Comment:
Next time, for function name, please follow the rule with Upper case first. For ex: ClearClient()